indico/modules/attachments/templates/_attachments.html

```diff
@@ -64,6 +64,14 @@
                    title="{% trans %}The access to this folder is restricted, regardless of the protection scheme of
                           the parent.{% endtrans %}"></i>
             {% endif %}
+            {% if folder.is_hidden and management%}
+                <i class=" icon-eye-blocked"
+                title="{% trans %}This folder is always hidden in the frontend.{% endtrans %}"></i>
+            {% endif %}
+            {% if folder.is_always_visible and management%}
+                <i class=" icon-eye"
+                title="{% trans %}This folder is always visible in the frontend.{% endtrans %}"></i>
```

Comment:
Maybe this is a bit noisy since it's enabled by default? Explicitly showing the hidden flag makes sense but for this one I'm not sure if there's any benefit

Reply:
Well, it is only shown in managment views. And for symmetry ( all other flags one can set on a folder are indicated) I think it is helpful.